src/stores/my-account.ts

```diff
@@ -179,4 +315,19 @@ export const useMyAccount = create<State & Actions>()((set, get) => ({
     }
     set({ isInitialized: true })
   },
+  getPrivateKey: async () => {
```

Comment:
so I also think this function is not needed, just call this in the `login` action, and everywhere else can just use the encodedSecretKey data.